templates/character_creator.html

```diff
@@ -0,0 +1,137 @@
+								<option>Security Drone</option>
+								<option>Opalite</option>
+								<option>Cyborg</option>
+							</select>
+						</div>
+						<div class="col-lg-4">
+							<label for="classname">Class</label>
+							<select name="classname" id="classname">
+								<option>Soldier</option>
+								<option>Specialist</option>
+								<option>Doctor</option>
+								<option>Engineer</option>
+								<option>Hacker</option>
+								<option>Pilot</option>
+								<option>Hunter</option>
+								<option>Gunslinger</option>
+								<option>Assassin</option>
+								<option>Berserker</option>
+							</select>
+						</div>
+						<div class="col-lg-4">
+							<label for="level"> Level </label>
+							<input class="form-control" type="text" name="level" id="level">
+						</div>
+					</div> <!-- endrow -->
+					<div class="form-group row">
+						<div class="col-lg-3">
+							<label for="str"> Strength </label>
+							<input class="form-control" type="text" name="" id="str">
+
+							<label for="per"> Perception </label>
+							<input class="form-control" type="text" name="" id="per">
+						</div>
+						<div class="col-lg-3">
+							<label for="str"> STR mod </label>
+							<input class="form-control" type="text" name="" id="str" disabled="true">
+
+							<label for="per"> PER mod</label>
+							<input class="form-control" type="text" name="" id="per" disabled="true">
+						</div>
+						<div class="col-lg-3">
+							<label for="fort"> Fortitude </label>
+							<input class="form-control" type="text" name="" id="fort">
+
+							<label for="cha"> Charisma </label>
+							<input class="form-control" type="text" name="" id="cha">
+						</div>
+						<div class="col-lg-3">
+							<label for="str"> FORT mod </label>
+							<input class="form-control" type="text" name="" id="str" disabled="true">
+
+							<label for="per"> CHA mod </label>
+							<input class="form-control" type="text" name="" id="per" disabled="true">
+						</div>
+					</div> <!-- endrow -->
+					<div class="form-group row">
+						<div class="col-lg-3">
+							<label for="int"> Intelligence </label>
+							<input class="form-control" type="text" name="" id="int">
+
+							<label for="dex"> Dexterity </label>
+							<input class="form-control" type="text" name="" id="dex">
+						</div>
+						<div class="col-lg-3">
+							<label for="str"> INT mod </label>
+							<input class="form-control" type="text" name="" id="str" disabled="true">
+
+							<label for="per"> DEX mod </label>
+							<input class="form-control" type="text" name="" id="per" disabled="true">
+						</div>
+						<div class="col-lg-3">
+							<label for="luck"> Luck </label>
+							<input class="form-control" type="text" name="Luck" id="luck">
+
+							<label for="totalSkill"> Total </label>
+							<input class="form-control" type="text" name="totalSkill" disabled="true">
+						</div>
+						<div class="col-lg-3">
+							<label for="str"> LUK mod </label>
```

Comment:
I've seen LUC, LUK, and LUCK at various points and various times. But since there's already precedent for the four-letter abbreviation in FORT, I think we should just go with LUCK.